kratos/solving_strategies/schemes/residual_based_bossak_displacement_scheme.hpp

```diff
@@ -435,14 +452,16 @@ class ResidualBasedBossakDisplacementScheme
     ///@{
 
     /**
-     * @brief The Generalized Alpha components
-     * @detail For more about it:
-     * J. Chung, G.M.Hubert. "A Time Integration Algorithm for Structural Dynamics with Improved Numerical Dissipation: The Generalized-α Method" ASME Journal of Applied Mechanics, 60, 371:375, 1993.
+     * @brief The Bossak Alpha components
      */
-    struct GeneralizedAlphaMethod
+    struct BossakAlphaMethod
     {
-        double f; /// Alpha Hilbert
-        double m; /// Alpha Bosssak
+        double alpha; /// Alpha Bossak
```

Comment:
Did you try using https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/time_discretization.h#L81?

Reply:
I think using this would require a proper refactoring of the internals, whereas this PR is only a cleanup

Reply:
In principle I agree but I would leave such relatively big changes to a separate PR